css/main.css

```diff
@@ -6,6 +6,44 @@
   box-sizing: border-box;
 }
 
+#close-btn {
+  z-index: 3;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+}
+
+.mobile-nav {
+  position: absolute;
+  background-color: #6070ff;
+  top: 0;
+  z-index: 2;
+  height: 100vh;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.d-none {
+  display: none;
+}
+
+.menu-items {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+}
```

Comment:
- [ ] The mobile view menu is not centralized as per the design. Kindly fix that. You can add an align-items property to the menu-items class and set it's value to center. Try as much as possible to stick to the design, as in the real world, you will have to stick to the clients specifications(design). See screenshot: 
![Screenshot from 2021-09-03 15-06-10](https://user-images.githubusercontent.com/65852621/132055302-f04fcf82-7fe2-48d9-bd7c-bfd03a9c7201.png)